Questions.tex

```diff
@@ -236,7 +236,7 @@ \section{Espérance variable aléatoire réelle}
 	i.e. si $\mathbb{E} (|X|^p) < \infty$, alors $\mathbb{E} (|X|^q) < \infty$ pour tout $q \in [0, p]$.
 	\begin{proof}
 		Si $0 \leq p \leq r$, on a les inégalités suivantes entre variables aléatoires positives:
-		$$|X|^p \leq \mathscr{1}_{\{|X|\leq 1\}} + |X|^r\mathscr{1}_{\{|X|>1\}} \leq 1 + |X|^r$$
+		$$|X|^p \leq \mathbb{1}_{\{|X|\leq 1\}} + |X|^r\mathbb{1}_{\{|X|>1\}} \leq 1 + |X|^r$$
```

Comment:
Incorrect rendering: 
![image](https://github.com/Yag000/PR5/assets/80483460/8cf8a1df-77f2-406f-9ce5-fcbf4b385698)
The correct one is `\mathds`
```suggestion
		$$|X|^p \leq \mathds{1}_{\{|X|\leq 1\}} + |X|^r\mathds{1}_{\{|X|>1\}} \leq 1 + |X|^r$$
```

Reply:
ok I just corrected thank you